docs/truecolor.md

```diff
@@ -0,0 +1,176 @@
+
+# Truecolor Over SSH With Wish
+
+## Color profiles? What, like it's hard?
+
+In an SSH session, the client only sends the `TERM` environment variable, which
+can only detect `256 color` support. If you run `echo $COLORTERM` in your shell
+you'll likely see `truecolor` as the result, which is what you want for ultra
+colorful terminal outputs. If you don't see that as a result, it might be time
+to try a new [terminal emulator][supported-emulators]. 
+
+Unfortunately, there is no standard way for terminals to detect `truecolor`
+support in an SSH session, hence why it defaults to `256 color`. Most terminals
+express their `truecolor` support with the `COLORTERM` environment variable,
+but this doesn't get sent when connecting over SSH. One workaround is to make
+SSH send this environment variable using `SendEnv COLORTERM` in the [`ssh
+config`][truecolor-ssh]. By default, the OpenSSH client (what you're using when
+you run `ssh`) will only send the `TERM` to the remote, so other variables must
+be configured. In the future, we hope to solve this problem by querying the
+terminal for support if `COLORTERM` is not detected.
+
+> [!NOTE]
+> Wish uses [termenv][termenv] under the hood, which will set truecolor for a few
```

Comment:
I don't even know if this sentence makes sense 😂 may need a lil tidy later

Reply:
I think the chain of deps is more like wish -> lipgloss -> termenv

Reply:
going to leave this part out for now given we're going to intro term changes soon which might include different default behaviour